src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -118,9 +120,20 @@ internal partial class InlineRenameSession : ForegroundThreadAffinitizedObject,
             _baseSolution = _triggerDocument.Project.Solution;
             this.UndoManager = workspace.Services.GetService<IInlineRenameUndoManager>();
 
+            this._editAndContinueWorkspaceService = workspace.Services.GetService<IEditAndContinueWorkspaceService>();
+            this._editAndContinueWorkspaceService.BeforeDebuggingStateChanged += OnBeforeDebuggingStateChanged;
+
             InitializeOpenBuffers(triggerSpan);
         }
 
+        private void OnBeforeDebuggingStateChanged(object sender, DebuggingStateChangedEventArgs args)
```

Comment:
Every stepping in the debugging mode would trigger this one with setting after to be Run. Do we have to call Commit() for all or we don't reach here after the renaming session is done?